runtime/rasdump/dmpsup.c

```diff
@@ -577,7 +577,42 @@ configureDumpAgents(J9JavaVM *vm)
 		}
 		if ( IS_CONSUMABLE(j9vm_args, xdumpIndex) && !IS_CONSUMED(j9vm_args, xdumpIndex) )
 		{
-			GET_OPTION_VALUE(xdumpIndex, ':', &optionString);
+			BOOLEAN isMappedToolDump = FALSE;
+			/* Handle mapped tool dump options */
+			if (HAS_MAPPING(j9vm_args, xdumpIndex)) {
+				char *mappingJ9Name = MAPPING_J9NAME(j9vm_args, xdumpIndex);
+				char *toolString = ":tool";
```

Comment:
Since the trailing colon is required (for the +1 later), it should be added here.